docs/modules/manual/pages/zoeken.adoc

```diff
@@ -0,0 +1,28 @@
+= Hoe gebruik ik de zoekfunctie in Solid CRS?
+:description: Een gebruikershandleiding voor het gebruiken van de zoekfunctie in Solid CRS.
+:sectanchors:
+:url-repo: https://github.com/netwerk-digitaal-erfgoed/solid-crs
+:imagesdir: ../images
+
+Dit document toont je hoe je de objecten van je erfgoedinstelling beheert in het Solid Collectieregistratiesysteem. 
+
+
+=== Vereisten
+Om de zoekfunctie te gebruiken als bezoeker zijn er geen vereisten. Indien je als beheerder de applicatie bezoekt, moet je je xref:authenticeer.adoc[aanmelden]. 
```

Comment:
Perhaps better to differentiate between solid-crs/presentatie as opposed to user/admin